CHANGELOG.md

```diff
@@ -1,51 +1,57 @@
 # Changelog
 
+## 1.0.0
+
+### Breaking Change
+
+- This version adds extra random digits to the URL used by the Twilio Video Apps. The format of the URL has changed from `https://video-app-xxxx-dev.twil.io` to `https://video-app-xxxx-yyyy-dev.twil.io`. The iOS and Android video apps must be upgraded in order to work with this new format. The React app does not need to be upgraded.
+
 ## 0.1.6
 
 ### Maintenance
 
-* Move @oclif/plugin-help from devDependencies to dependencies to resolve NPM warning.
```

Comment:
Nit, but were these formatting changes intentional on the older versions in the changelog?

Reply:
Yeah. This is the work of our automatic formatter (prettier). I don't think I had it enabled previously.